inference-engine/src/gna_plugin/gna_device.cpp

```diff
@@ -62,6 +63,8 @@ uint32_t GNADeviceHelper::propagate(const intel_nnet_type_t *pNeuralNetwork,
     return reqId;
 }
 #else
+std::mutex GNADeviceHelper::acrossPluginsSync{};
```

Comment:
should not it be marked as `static` or put into unnamed namespace?

Reply:
Its declared as `static` class member so no need put `static` here in initialization.

Reply:
oh, I see. I thought those are different variables..